registration-processor/registration-processor-registration-status-service-impl/src/main/java/io/mosip/registration/processor/status/service/impl/RegistrationStatusServiceImpl.java

```diff
@@ -72,6 +74,10 @@
 	/** The reg proc logger. */
 	private static Logger regProcLogger = RegProcessorLogger.getLogger(RegistrationStatusServiceImpl.class);
 
+	/** The sync registration service. */
+	@Autowired
+	SyncRegistrationService<SyncResponseDto, SyncRegistrationDto> syncRegistrationService;
```

Comment:
do we need this autowiring and imports ?

Reply:
Fixed

Reply:
Still the imports are there